README.md

```diff
@@ -80,29 +88,34 @@ routes:
 
 topics:
     hello_ros2: { type: "std_msgs/String", route: dds_to_ros2 }
-    hello_fiware: { type: "std_msgs/String", route: ros2_to_dds }
+    hello_dds: { type: "std_msgs/String", route: ros2_to_dds }
 ```
 
 To see how general SOSS systems, users and topics are configured, please refer to [SOSS' documentation][soss].
 
 For the DDS system handle the user must add two extra YAML maps to the configuration file, which are “dynamic types” and “participant”, as seen above.
 
-On one hand, the dynamic types map tells the DDS system handle how a certain type is mapped. 
-This is necessary to convert the type from a soss message, which is the type used inside soss, to a dynamic type, which is the type used in DDS. 
+On one hand, the dynamic types map tells the DDS system handle how a certain type is mapped.
+This is necessary to convert the type from a soss message, which is the type used inside soss, to a dynamic type, which is the type used in DDS.
 This conversion is done dynamically at runtime.
 
 To have a guide on how dynamic types are defined in YAML files, see the [YAML dynamic types](#yaml-dynamic-types) section.
 
-*The dynamic types standard does not allow certain characters in its names*. 
-For this reason, if a type defined in the topics section of the configuration file has in its name a `/`, the dds system handle will map that character into two underscores. 
-That's why the type inside the dynamic types map is std_msgs__String, while the type inside the topics section is std_msgs/String. This is *something important to notice when connecting to ROS2*, because in ROS2 most of the types have a `/` in their names. 
+*The dynamic types standard does not allow certain characters in its names*.
```

Comment:
I know this is something I did, but here instead of _cursive_ I intended to put it in **bold**. Would you change that for me, to avoid creating a new pull request, please?